vela-templates/definitions/usage-examples/application-with-startup-probe.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: core.oam.dev/v1beta1
```

Comment:
please add the example here align the format: https://github.com/kubevela/kubevela/tree/master/references/docgen/def-doc/trait

They'll be generated as reference doc automatically. https://kubevela.net/docs/end-user/traits/references